From 1648648b35e32a859a4f6e74a9009d3e5fa4cdfa Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Thu, 6 Sep 2018 15:09:17 +0300
Subject: [PATCH 290/706] enetc: Add ethtool link settings support

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
(cherry picked from commit 067bdbe2d25dadace96b22c2d8c81aa75c897cb0)
(cherry picked from commit cae71b5f995ad45582631c1f10b788409bf4755b)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/freescale/enetc/enetc_ethtool.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/net/ethernet/freescale/enetc/enetc_ethtool.c b/drivers/net/ethernet/freescale/enetc/enetc_ethtool.c
index 93a60d1f3fe8..9ec42f9591f4 100644
--- a/drivers/net/ethernet/freescale/enetc/enetc_ethtool.c
+++ b/drivers/net/ethernet/freescale/enetc/enetc_ethtool.c
@@ -617,6 +617,8 @@ const struct ethtool_ops enetc_pf_ethtool_ops = {
 	.set_rxfh = enetc_set_rxfh,
 	.get_ringparam = enetc_get_ringparam,
 	.get_ts_info = enetc_get_ts_info,
+	.get_link_ksettings = phy_ethtool_get_link_ksettings,
+	.set_link_ksettings = phy_ethtool_set_link_ksettings,
 };
 
 const struct ethtool_ops enetc_vf_ethtool_ops = {
-- 
2.17.1

